### ISSUE TITLE
Memory leak when calling run_script

### ISSUE THREAD

**Issue by Apprisco** [spyoungtech/ahk#338]
### describe your issue

Had a python based AHK script running that consistently presses some values using asyncahk.
Memory kept increasing over time, currently over 4GB.``

### ahk.__version__

latest

### AutoHotkey version

v1

### Code to reproduce the issue

```python
from ahk import AHK,AsyncAHK

class Trove:
    ahk=AHK(directives=[NoTrayIcon])
    asyncAHK=AHK(directives=[NoTrayIcon])

while True:
    script='''.......'''
    ahk.run_script(script)
    asyncAHK.run_script(script)
```

### Traceback/Error message

Observe memory usage increasing over time.

**Comment by spyoungtech** [comment]
Thanks for the report. I confirmed this behavior is reproducible with `run_script` and calls with `blocking=False`. It's likely related to resources for the subprocess.

**Comment by spyoungtech** [comment]
I believe I found the cause of this.

The creation of AutoHotkey processes uses the [start method](https://github.com/spyoungtech/ahk/blob/46bce5c3e19aaa9fe49f1a90816a80a96a39a2c2/ahk/_async/transport.py#L133-L136) (or [here](https://github.com/spyoungtech/ahk/blob/46bce5c3e19aaa9fe49f1a90816a80a96a39a2c2/ahk/_sync/transport.py#L122-L125) in the sync API), which includes a call to `atexit.register`, which is an attempt to ensure that the subprocess will be cleaned up when the Python script is finished -- to avoid leaving AutoHotkey running after Python is done executing.

However, the obviously causes a reference to be held for the `subprocess.Popen` object, so it never gets garbage collected.

When I remove this line, the memory buildup no longer occurs.

**Comment by Apprisco** [comment]
How am I able to copy your fix? Will there be an updated version of AHK wrapper released soon? Thanks :)

**Comment by spyoungtech** [comment]
@Apprisco I've opened #339 with a fix for this issue. As long as tests pass, it should be released within the next day, if not the next hour or so.

**Comment by spyoungtech** [comment]
This is fixed in version 1.8.0

**Comment by Apprisco** [comment]
Hell yeah, thanks man!
